modules/server/src/main/ts/bedrock/auto/Driver.ts

```diff
@@ -67,6 +68,14 @@ const addArguments = (capabilities: Record<string, any>, name: string, args: str
   capabilities[name].args = currentArgs.concat(args);
 };
 
+const getExtraBrowserCapabilities = (settings: DriverSettings): string[] => {
+  if (settings.extraBrowserCapabilities.length > 0) {
```

Comment:
Should this be trimmed before comparing? As if it's just an empty string then you're not going to want a `[' ']` array to be returned.

Reply:
Good point.

- [ ] trim before comparing.